src/Microsoft.IdentityModel.Tokens/X509CertificateHelper.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// Helper class to load X509Certificate2 from byte array.
+    /// </summary>
+    internal class X509CertificateHelper
+    {
+        public static X509Certificate2 Load(byte[] data)
```

Comment:
nit: What about naming class more generic like `CertificateHelper` and method `LoadX509Certificate`? That way we can add more methods here to load different cert types in the future, if needed (otherwise, we'd have to create a separate class for each cert type). And since this is an internal class that's used in multiple projects, we should make sure the API is right, since we won't be able to change it.

Reply:
@keegan-caruso Thoughts on adding an internal class to be used in multiple projects? I suppose since we'll add the public API analyzers, this is okay. You mentioned previously an approach of create a separate duplicate helper class in each project to avoid the issues with internal friendlies.

Reply:
I was considering making a separate helper class per project too.. happy to do that. Also happy to name it a bit more generic + .LoadX509()

Reply:
I agree with @pmaytak a more general name may be an advantage.
Is there a reason why this is internal?

Reply:
Spoke with @keegan-caruso - we'll use [compile include ](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/f0d09d408948c6920f171299bee9167d0850a51b/src/Microsoft.IdentityModel.Logging/Microsoft.IdentityModel.Logging.csproj#L20C1-L22C15) instead of internals visible to. This way we avoid the issues with internals but we keep the helper class in one spot for maintainability.

@brentschmaltz we want to keep this internal. This is a helper class that we don't want to ship.

Reply:
Done